src/app/globals.css

```diff
@@ -10,20 +10,23 @@ body {
   overflow-x: hidden;
 }
 
-body {
+/* body {
   color: rgb(var(--foreground-rgb));
```

Comment:
it was the previous CSS that was causing the weird black and white bands across index. I commented it to avoid deleting during testing. It could be removed now